open_schools_platform/user_management/users/selectors.py

```diff
@@ -43,3 +43,12 @@ def get_token_with_checks(key: str, verify_check: bool = True, is_alive_check: b
     if is_alive_check and not is_token_alive(token):
         raise AuthenticationFailed(detail="Token is overdue.")
     return token
+
+
+def get_firebase_token_entity(*, filters=None) -> FirebaseToken:
+    filters = filters or {}
+
+    qs = FirebaseToken.objects.all().order_by('created_at')
+    token = FirebaseTokenFilter(filters, qs).qs.last()
+
+    return token
```

Comment:
okay but we can get FirebaseToken with user instance like that: user.firebase_token